frontend/src/main/web/lib/constants/Views.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Router from 'react-router';
+import StringUtils from '../utils/StringUtils';
+import RootContent from '../components/RootContent';
+import UserProfile from '../components/UserProfile';
+import SystemGlossary from '../components/SystemGlossary';
+
+var Views = {
+  USER_PROFILE: 0,
+  GLOSSARY: 1,
+
+  getView: function(value) {
+    if(StringUtils.isEmptyOrNull(value)) {
+      return null;
+    }
+
+    switch (value.toLowerCase()) {
+      case 'profile':
+      case 'user_profile':
+      case 'userProfile':
```

Comment:
Are all these options to interact with legacy code? If they are not, we should just pick one. If someone wants to find all times the user profile view is used, they should be able to do a single search like `.getView('profile')`, not 3 different searches.

Also, `'userProfile'` will never match anything that is `.toLowerCase()`.

If possible, change this to use `views.USER_PROFILE` instead of a raw string, otherwise at least try to make it just one option.